src/server/pps/server/s3g_sidecar_test.go

```diff
@@ -666,14 +664,14 @@ func TestS3SkippedDatums(t *testing.T) {
 		pipelineCommit := client.NewCommit(pipeline, "master", "")
 		// Add files to 'repo'. Old files in 'repo' should be reprocessed in every
 		// job, changing the 'background' field in the output
-		for i := 0; i < 10; i++ {
+		for i := 1; i < 7; i++ {
```

Comment:
Going to file an issue here. It looks like by the later iterations it take about 50 seconds for a job to run. This seems like a lot to me. Maybe there's some big latency each time we query the S3 endpoint, given that there are 3 of these that get run per Datum, and there are approx 10 datums at the end?